flybywiresim/a32nx
# refactor(efb): few fixes and restructuring 🥚

## Summary of Changes

fix brightness for leaking and aliased font, fix wrong status bar time, refactoring & file restructure.


Discord username (if different from GitHub): myselfaaryan#2092

## Testing Instructions

1. Boot the EFB
2. Check if loading screen works perfectly.
3. Check if status bar has the correct zulu time.
4. Check if font is less leaking and less bold.
5. Check if icon is present next to weather ICAO.
6. Check if degree sign is aligned perfectly.



## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
